.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+exclude: 'build|src/deps|src/obsolete'
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.2.3
+    hooks:
+    -   id: trailing-whitespace
+    -   id: check-added-large-files
+        args: ['--maxkb=2000']
+    -   id: end-of-file-fixer
+
+-   repo: https://github.com/asottile/seed-isort-config
+    rev: v1.9.2
+    hooks:
+    -   id: seed-isort-config
+
+-   repo: https://github.com/pre-commit/mirrors-isort
+    rev: v4.3.20
+    hooks:
+    -   id: isort
+        language_version: python3.6
```

Comment:
I don't believe these are language versions are needed and they prevent you from using python3.7